src/emitters/emitter.sqns.ts

```diff
@@ -92,6 +89,30 @@ export class SqnsEmitter implements IEmitter {
     await this.createEventSourceMappings();
   }
 
+  private validateAwsConfig() {
+    if (!this.options.awsConfig) {
+      throw new Error(
+        `awsConfig is required in options when using external broker.`
+      );
+    } else {
+      const { region, accountId } = this.options.awsConfig;
+      if (!accountId)
+        this.getAwsAccountId(region).then((accountId) => {
+          this.options.awsConfig!.accountId = accountId;
+        }).catch((error) => {
+          console.error(error.message);
+          throw error;
+        });;
+    };
+  }
+
+  private async getAwsAccountId(region: string): Promise<string> {
+   const client = new STSClient({ region });
+   const accountId = (await client.send(new GetCallerIdentityCommand({}))).Account;
+   if (!accountId) throw new Error('No Account Id Found.');
+   return accountId;
```

Comment:
```suggestion
   if (!accountId) {
   throw new Error('No Account Id Found.');
   return accountId};
```

Reply:
How even this is possible to return Id after throwing error.

Reply:
```suggestion
   if (!accountId) 
   {
      throw new Error('No Account Id Found.');
   }
   return accountId;
```
is what I meant